docs/coverage-reporter/alternative-ways-of-running-coverage-reporter.md

```diff
@@ -6,6 +6,9 @@ description: There are alternative ways of running or installing Codacy Coverage
 
 The following sections list the alternative ways of running or installing Codacy Coverage Reporter.
 
+!!! important
+    **If you're using Codacy Self-hosted** you must use a Codacy Coverage Reporter version that's compatible with your Codacy instance. To do this, specify the Codacy Coverage Reporter version in the format `self-hosted-<major.minor.patch>`, where `<major.minor.patch>` is the version of your Codacy instance.
```

Comment:
the `self-hosted-...` tags aren't working as valid versions anymore 😞 
does it make sense to merge thi note anyway?

Reply:
As a first solution, for now, I'm thinking of removing this part and including a hard-coded version on the main setup instructions.

I'm going to push a commit with these changes and ask for another review.